### ISSUE TITLE
[Vanilla] Missing incantations: Aspects of the Crucible Breath and Horns + typo fix

### ISSUE THREAD

**Issue by ashemedai** [Gobluebro/Elden-Ring-Checklist#63]
`data/lists/spells.ts` misses these incantations:

```ts
      {
        id: "d87d0166-3de2-4692-b299-6a7e803c4f88",
        description: rawHTMLLink(
          "https://eldenring.wiki.fextralife.com/Aspects+of+the+Crucible:+Breath",
          "Aspects of the Crucible: Breath"
        ),
      },
      {
        id: "c74ca270-f12d-4aca-b418-cfb830a05ea9",
        description: rawHTMLLink(
          "https://eldenring.wiki.fextralife.com/Aspects+of+the+Crucible:+Horns",
          "Aspects of the Crucible: Horns"
        ),
      },
```

Typo fix for `data/lists/quests.ts` needed (aspect to aspects):

```ts
      {
        id: "e4b0b2fc-7a40-4af3-94b6-1e65db9e3eac",
        description:
          "Tanith's Knight will invade. Kill and loot the Knight for Aspects of the Crucible: Breath.",
      },
```

**Comment by Gobluebro** [comment]
Thank you for catching these!

**Comment by Gobluebro** [comment]
> Typo fix for `data/lists/quests.ts` needed (aspect to aspects):
> 
> ```ts
>       {
>         id: "e4b0b2fc-7a40-4af3-94b6-1e65db9e3eac",
>         description:
>           "Tanith's Knight will invade. Kill and loot the Knight for Aspects of the Crucible: Breath.",
>       },
> ```
Fixed in 7c2575c

**Comment by Gobluebro** [comment]
> `data/lists/spells.ts` misses these incantations:
> 
> ```ts
>       {
>         id: "d87d0166-3de2-4692-b299-6a7e803c4f88",
>         description: rawHTMLLink(
>           "https://eldenring.wiki.fextralife.com/Aspects+of+the+Crucible:+Breath",
>           "Aspects of the Crucible: Breath"
>         ),
>       },
>       {
>         id: "c74ca270-f12d-4aca-b418-cfb830a05ea9",
>         description: rawHTMLLink(
>           "https://eldenring.wiki.fextralife.com/Aspects+of+the+Crucible:+Horns",
>           "Aspects of the Crucible: Horns"
>         ),
>       },
> ```
Fixed in c7cc580
